RecoMuon/MuonIdentification/python/muons1stStep_cfi.py

```diff
@@ -20,7 +20,7 @@
 
     fillEnergy = cms.bool(True),
     # OR
-    maxAbsPullX = cms.double(4.0),
+    maxAbsPullX = cms.double(3.0),
```

Comment:
this will probably be noticeable in B=0T setup as a loss of 1/4 at lower p values. IIRC default track p in B=0T is 5 GeV. So, below this value there will be some signal loss.
... probably OK.